core/services/pipeline/runner.go

```diff
@@ -215,23 +215,23 @@ func (r *runner) processTaskRun() (anyRemaining bool, err error) {
 			logger.Errorw("Pipeline task run could not be unmarshaled", append(loggerFields, "error", err)...)
 			return Result{Error: err}
 		}
-		var job models.JobSpecV2
-		err = txdb.Find(&job, "id = ?", jobID).Error
+		var spec Spec
+		err = txdb.Find(&spec, "id = ?", taskRun.PipelineRun.PipelineSpecID).Error
```

Comment:
Was this broken before with `jobID`?

Reply:
nah - just removing the dependency on jobs and using the pipeline spec instead